tests/ml_commons/test_model_access_control.py

```diff
@@ -0,0 +1,252 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import os
+import time
+from unittest.mock import patch
+
+import pytest
+from opensearchpy.exceptions import NotFoundError, RequestError
+from packaging.version import parse as parse_version
+
+from opensearch_py_ml.ml_commons.model_access_control import ModelAccessControl
+from tests import OPENSEARCH_TEST_CLIENT
+
+OPENSEARCH_VERSION = parse_version(os.environ.get("OPENSEARCH_VERSION", "2.7.0"))
+MAC_MIN_VERSION = parse_version("2.8.0")
```

Comment:
What did we mean by `MAC` here?

Reply:
Model Access Control. wanted to keep it short. I'll add a comment for reference.